libcrux-ml-kem/benches/ml-kem.rs

```diff
@@ -44,6 +44,41 @@ pub fn key_generation(c: &mut Criterion) {
     init!(mlkem512, "Key Generation", c);
     init!(mlkem768, "Key Generation", c);
     init!(mlkem1024, "Key Generation", c);
+
+    #[cfg(all(
+        feature = "mlkem768",
+        feature = "pre-verification",
+        feature = "simd256"
+    ))]
+    c.bench_function("libcrux avx2 unpacked (external random)", |b| {
```

Comment:
All of these should use the macros in here, then they become only a few lines. But we can clean that up later if you prefer and file an issue.

Reply:
Let's do that later. I did the rest.